css/card.css

```
@@ -0,0 +1,50 @@
+.grid{
+    display: flex;
+    justify-content: space-evenly;
+    flex-wrap: wrap;
+}
```

```suggestion
.cards-collection-wrapper{
    display: flex;
    justify-content: space-evenly;
    flex-wrap: wrap;
}
```
Hey Kunal, we can change the class name from "grid" to something more relatable. I have made a suggestion above. Also, since you are using flexbox inside it,  class name "grid" can cause confusions. 
Let me know what you think!